fix: update widget to ref 'attribute' not 'attributeName'

agdavid opened the pull request:
- update required prop as `attribute` instead of `attributeName` in widgets to avoid errors in instant-search demo project
- update `package.json` dependencies to react-instantsearch ^5.0.3
- update theme CDN to new `instantsearch.css` per documentation

Haroenv commented:
Thanks David, this looks good, I guess it happened because we have `latest` in package.json here, which seems like an odd choice. Feel free to change that in this PR to `^5.0.3`, maybe also with some theme changes?

agdavid commented:
Thanks @Haroenv for the review. I updated the dependencies for react-instantsearch to ^5.0.3 per your suggestion. Also changed the CDN for the theme to the new instantsearch.css referenced in the [migration guide](https://community.algolia.com/react-instantsearch/guide/Migration_guide_v5.html). Happy to pull back the last commit on the theme CDN if that's not what you were suggesting.
